tests/containers/rootless_podman.pm

```diff
@@ -36,14 +36,18 @@ sub run {
 
     install_podman_when_needed($host_distri);
     allow_selected_insecure_registries(runtime => $runtime);
-    my $user      = $testapi::username;
-    my $check_msg = 'Checking allocation range of user';
-    script_run "grep $user /etc/subuid || echo /etc/subuid has no uid range for $user", output => $check_msg;
-    script_run "grep $user /etc/subgid || echo /etc/subgid has no gid range for $user", output => $check_msg;
-    assert_script_run "usermod --add-subuids 200000-201000 --add-subgids 200000-201000 $user";
+    my $user         = $testapi::username;
+    my $subuid_start = get_user_subuid($user);
+    if ($subuid_start eq '') {
+        $subuid_start = 200000;
+        my $subuid_range = $subuid_start + 1000;
+        assert_script_run "usermod --add-subuids $subuid_start-$subuid_range --add-subgids $subuid_start-$subuid_range $user";
+    }
+    else {
+        record_soft_failure 'bsc#1179261 - YaST creates incomplete user accounts';
```

Comment:
The YaST bug causes `$subuid_start` to be empty, so this belongs inside the `if`